spec/requests/api/v1/timelines/link_spec.rb

```diff
@@ -17,6 +17,14 @@
     end
   end
 
+  # The default settings are that timeline_preview_local is true but
+  # timeline_preview_remote is false, which caused this spec to fail because it
+  # assumes the default visibility is true.
+  before do
+    Form::AdminSettings.new(timeline_preview_local: true).save
+    Form::AdminSettings.new(timeline_preview_remote: true).save
+  end
```

Comment:
We probably need a way on `LinkFeed` to say "exclude remote statuses", since it surfaces both remote and local links, which could allow a bad acting remote server to poison your link timeline with harmful links if auto-acceptance of trending for links is enabled